src/main/java/io/sealights/plugins/sealightsjenkins/integration/JenkinsPomFile.java

```diff
@@ -48,4 +50,15 @@ protected Document getDocumentInternal() throws IOException, InterruptedExceptio
         Document doc = fp.act(new GetDocumentFileCallable());
         return doc;
     }
+
+    @Override
+    //TODO implement
+    public void backup() throws Exception {
+        String backupFile = this.filename + BACKUP_EXTENSION;
+        log.info("JenkinsPomFile - creating a back up file: " + backupFile);
+        VirtualChannel channel = Computer.currentComputer().getChannel();
```

Comment:
Should we clean that using the CleanupManager?

Reply:
yup, and it cleaned outside